packages/dataviews/src/stories/dataform.story.tsx

```diff
@@ -1441,6 +1457,15 @@ const LayoutCardComponent = ( { withHeader }: { withHeader: boolean } ) => {
 						withHeader: false,
 					},
 				},
+				{
+					id: 'important-information',
+					label: 'Important information',
+					layout: {
+						type: 'card',
+						isCollapsible: false,
```

Comment:
It'd be nice to have this as an argument to the story, like `withHeader` is, to act as an educational tool for readers.

Reply:
I agree, I added it and removed this new card. Here's how it looks:

https://github.com/user-attachments/assets/ce7308bc-7f7b-4b28-bc68-356ae65cfb36